app/boards/arm/nucleo_f446re/nucleo_f446re.dts

```diff
@@ -19,7 +19,8 @@
 		zephyr,console = &usart2;
 		zephyr,shell-uart = &usart2;
 		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
+		zephyr,flash = &w25q128jvsiq;
+        zephyr,flash-controller = &w25q128jvsiq;
```

Comment:
Indent

Reply:
Not sure how to get rid of my changes to `app`, checking our or resetting many commits back doesn't do anything